compair/manage/score.py

```diff
@@ -0,0 +1,28 @@
+"""
+    Recalculate Scores
+"""
+
+from flask_script import Manager, prompt_bool
+from compair.models import Comparison, Assignment
+
+
+manager = Manager(usage="Recalculate Assignment Answer Scores")
+
+
+@manager.option('-a', '--assignment', dest='assignment_id', help='Specify a Assignment ID to generate report from.')
+def recalculate(assignment_id):
+    if prompt_bool("""All current answer scores and anwer criterion scores will be overwritten.
+Final scores may differ slightly due to floating point rounding (especially if recalculating on different systems).
+Are you sure?"""):
+        if not assignment_id:
+            raise RuntimeError("Assignment with ID {} is not found.".format(assignment_id))
+
+        assignment = Assignment.query.filter_by(id=assignment_id).first()
+        if not assignment:
+            raise RuntimeError("Assignment with ID {} is not found.".format(assignment_id))
```

Comment:
NIT: would be nice to put the parameter checking before the prompt.

Reply:
Good point, fixed